psscriptanalyzer/Backup-Database.psd1

```
@@ -0,0 +1,20 @@
+@{
+    RootModule        = 'Backup-Database.psm1'
+    ModuleVersion     = '0.1.0'
+    GUID              = 'a1b2c3d4-e5f6-7890-1234-567890abcdef'
+    Author            = 'Demoapp Platform Team'
+    CompanyName       = 'Demoapp'
+    Copyright         = '(c) 2026 Demoapp.'
+    Description       = 'Backup and restore helpers for Demoapp databases.'
```

PSScriptAnalyzer (1.25.0)

[warning] Use space before and after binary and assignment operators.

Suggested fix: Use space before and after binary and assignment operators.

(PSUseConsistentWhitespace)